bridgestyle/arcgis/constants.py

```diff
@@ -1,18 +1,23 @@
+import re
 from enum import Enum
 
 
 ESRI_SYMBOLS_FONT = "ESRI Default Marker"
-PT_TO_PX_FACTOR = 4/3
-POLYGON_FILL_RESIZE_FACTOR = 2/3
-OFFSET_FACTOR = 4/3
+PT_TO_PX_FACTOR = 4 / 3
+POLYGON_FILL_RESIZE_FACTOR = 2 / 3
+OFFSET_FACTOR = 4 / 3
+PROPERTY_NAME_PATTERN = re.compile(r"^[a-zA-Z_][a-zA-Z0-9_]*$")
```

Comment:
How restrictif should it be ? Currently accent will be not valid, as well as `-`.
If we want to support accent, that could be a solution: `^[\D][\w]*$`
Or with `-`  (after the first character): `^[\D][\w-]*$`
https://docs.python.org/fr/3/howto/regex.html

But it's harder to read, a small description could be great if you use this regexp.

Reply:
I cannot tell, as I am assuming you can have a variety of data sources when using a filter in a SLD. I didn't want to remove the check altogether, since it's probably been added for a reason (that I couldn't find). So I added this pattern which more or less correspond to a valid PostgreSQL column, since that's our use case.